compiler/rustc_parse/src/parser/nonterminal.rs

```diff
@@ -170,6 +175,7 @@ impl<'a> Parser<'a> {
                 NtPath(P(self.collect_tokens_no_attrs(|this| this.parse_path(PathStyle::Type))?))
             }
             NonterminalKind::Meta => NtMeta(P(self.parse_attr_item(true)?)),
+            NonterminalKind::Meta2021 => NtMeta(P(self.parse_attr_item_no_unsafe(true)?)),
```

Comment:
Again, I'm not sure this is necessary because using `unsafe(...)` currently fails to parse (see [playground example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=c728c2e4abf32a38d67ce89ab665b3f9)). So I think this should fall under the following paragraph from [RFC 3531](https://rust-lang.github.io/rfcs/3531-macro-fragment-policy.html):

> In cases where we’re adding new syntax and updating the grammar to include that new syntax, if we can update the corresponding fragment specifier simultaneously to match the new grammar in such a way that we do not risk changing the behavior of existing macros (i.e., because the new syntax previously would have failed parsing), then we will do that so as to prevent or minimize divergence between the fragment specifier and the new grammar. 

I.e. the question is: Can the addition of `unsafe(..)` change the behavior of any currently valid macros / macro invocations?